task-requests/README.md

```diff
@@ -6,34 +6,37 @@
     taskId: <string>
     externalIssueId: <string>
     requestType: ASSIGNMENT | CREATION
-    requestors : <Array<Requestors>>
+    users : <Array<users>>
     status : PENDING | APPROVED | DENIED
+    taskTitle: <string>
+    usersCount: <number>
     createdAt: <epoch>
     createdBy: <string>
     lastModifiedAt: <epoch>
-    lastModifiedBy: <epoch>
+    lastModifiedBy: <string>
 }
 ```
-#### Requestors Object:
+#### Users Object:
 ```
 {
 	userId: <string>
-	status: APPROVED | PENDING | NOT_ASSIGNED
+	status: APPROVED | PENDING 
 	proposedDeadline: <epoch>
+	proposedStartDate: <epoch>
+  description: <string>
 }
-
 ```
 
 ## **Requests**
 
 |               Route                |    Description    |
 | :--------------------------------: | :---------------: |
-|      [GET /task-requests](#get-task-requests)      | Return all task requests |
-|      [GET /task-requests/:id](#get-task-requests-id)      | Return task request details for the provided id |
-|     [POST /task-requests](#post-task-requests)     | Creates new task request  |
-|     [PATCH /task-requests/:id](#patch-task-requests-id) |   Updates task request   |
+|      [GET /taskRequests](#get-task-requests)      | Return all task requests |
+|      [GET /taskRequests/:id](#get-task-requests-id)      | Return task request details for the provided id |
+|     [POST /taskRequests](#post-task-requests)     | Creates new task request  |
+|     [PATCH /taskRequests](#patch-task-requests) |   Updates task request   |
```

Comment:
It is recommended to use either kebab case or snake case when naming your APIs, but camel case should be avoided.

Reply:
These api names were previously approved and present in backend repo before I started working on these. It should be fixed with this issue: 
- https://github.com/Real-Dev-Squad/website-backend/issues/1597